test/SILGen/objc_imported_generic.swift

```diff
@@ -38,7 +38,7 @@ public func genericSubscriptOnAnyObject(o: AnyObject, b: Bool) -> AnyObject? {
 
 // CHECK-LABEL: sil @_TF21objc_imported_generic27genericSubscriptOnAnyObjectFT1oPs9AnyObject_1bSb_GSqPS0___
 // CHECK: bb0([[ANY:%.*]]
-// CHCEK:   [[OPENED_ANY:%.*]] = open_existential_ref [[ANY]]
+// CHECK:   [[OPENED_ANY:%.*]] = open_existential_ref [[ANY]]
```

Comment:
@jtbandes Do you develop Swift on a Mac? If so, could you investigate the correct way to handle that? Simply changing to `CHECK` makes the Mac buildbot sad :-(